bootstrap-app/src/cmr/bootstrap/api/routes.clj

```diff
@@ -31,7 +31,8 @@
    [ring.middleware.json :as ring-json]
    [ring.middleware.keyword-params :as keyword-params]
    [ring.middleware.nested-params :as nested-params]
-   [ring.middleware.params :as params]))
+   [ring.middleware.params :as params]
+   [cmr.common-app.api.request-logger :as req-log]))
```

Comment:
This should be moved up in alphabetical order.

Reply:
I also ran condo on the file and cleaned up other things.